src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1063,7 +1063,6 @@ void RaftPart::processAppendLogResponses(const AppendLogResponses& resps,
       */
       auto [code, lastCommitId, lastCommitTerm] = commitLogs(std::move(walIt), true, true);
       if (code == nebula::cpp2::ErrorCode::SUCCEEDED) {
-        stats::StatsManager::addValue(kCommitLogLatencyUs, execTime_);
```

Comment:
Could add the `kCommitLogLatencyUs` in the scope from line 1054 to 1082?

Reply:
There are two places that will call commitlogs, so if you only add it to the place you said, then the commit latency will record less data. If you add it to both places, then the meaning of this metric will be unclear

Reply:
How about add it in `commitLogs` itself, that could be unified.

Reply:
In this PR, it is modified in this way

Reply:
👌 my bad...